pyproject.toml

```diff
@@ -23,6 +23,7 @@ async-lru = "^2.0.4"
 fsspec = "^2024.2.0"
 s3fs = "^2024.2.0"
 httpx = "^0.26.0"
+fastapi-utilities = "^0.2.0"
```

Comment:
sorry, one more, :joy:

Reply:
and probably need to update the lock file as well

Reply:
Curses!  removed